src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -72,6 +72,16 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
   const addDataFeatures = getFeaturesByCategory(FeatureCatalogueCategory.DATA);
   const manageDataFeatures = getFeaturesByCategory(FeatureCatalogueCategory.ADMIN);
   const devTools = findFeatureById('console');
+  const noDataConfig: KibanaPageTemplateProps['noDataConfig'] = {
+    solution: 'Analytics',
```

Comment:
Does this need to run through i18n? (I don't know the answer to this as I'm not that familiar with the component)

Reply:
Good point. I'll see how others are doing their empty state pages.

Reply:
These get handled at the template level:

https://github.com/elastic/kibana/blob/f952643e542a712149883d03f3cc99584d2b17eb/src/plugins/kibana_react/public/page_template/no_data_page/no_data_page.tsx#L146-L166